website/pages/docs/billing-and-usage/overview.md

```diff
@@ -7,30 +7,34 @@ description: Learn about CloudQuery's billing model
 
-Plugins are split in two categories: API plugins and Database plugins. Each category has its own price per row and the price goes down the more you sync within a month. See the [Pricing](https://www.cloudquery.io/pricing) page for the actual prices.
+Plugins are divided into two categories: API plugins and Database plugins. Each category has a price per row, and the price decreases the more you sync within a month. For the actual prices, see the [Pricing](https://www.cloudquery.io/pricing) page.
 
-Some source plugins are [free](https://hub.cloudquery.io/plugins/source?tiers=free) to use without charges. Similarly, your own plugins created with CloudQuery SDK are free to use.
+Some source plugins are [free](https://hub.cloudquery.io/plugins/source?tiers=free) and can be used without charges. Similarly, your plugins created with CloudQuery SDK are free to use.
 
-Some plugins (such as AWS, GCP, or Azure) have some tables that are free to sync. These tables usually contain static metadata that does not change often and have large amount of rows. These tables are marked `Free` in the individual plugin documentation.
+Some plugins (such as AWS, GCP, or Azure) have tables that are free to sync. These tables usually contain static metadata that does not change often and have a large number of rows. They are marked `Free` in the individual plugin documentation.
 
 ### Additional charges for cloud syncs
 
-When running syncs with CloudQuery Cloud, you may incur additional costs in the form of Egress, vCPU, and vRAM.
+When running syncs with CloudQuery Cloud, you may incur additional costs in Egress, vCPU, and vRAM.
 
 ### Monitoring your usage
 
-You can see your exact consumption in CloudQuery Cloud's billing page with details on the rows synced with individual plugins broken down by day, and how much of the free quota you have used in the current month.
+You can see your exact consumption in CloudQuery Cloud's billing page, which includes details on the rows synced with individual plugins broken down by day and how much of the free quota you used in the current month.
 
 ![Usage chart](/images/docs/billing-and-usage/usage-chart.png)
 
-If you used cloud syncs, your egress, vCPU, and vRAM consumption will be displayed at the bottom of the page.
+If you use cloud syncs, your egress, vCPU, and vRAM consumption will be displayed at the bottom of the page.
 
 ![Cloud syncs usage](/images/docs/billing-and-usage/cloud-syncs.png)
+
+## AWS Marketplace
+
+You can also purchase CloudQuery directly through the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-lowyuyay5a37s#pdp-pricing). This option streamlines billing and usage management, seamlessly integrating CloudQuery into your existing AWS environment. Explore the pricing and get started with CloudQuery on the AWS Marketplace today.
```

Comment:
The usage management within the AWS Marketplace is not a feature we want to highlight. It is super difficult to understand usage in the AWS marketplace. The Cost & Explorer Explorer that is intended for understanding usage takes hours to update.

Reply:
I hear you - we do not want to position it as the best way to use CQ. I added it to the bottom of the billing section, but how would you feel if I softened the language a bit? I still think it's important to mention that the AWS marketplace is an option. Something short and simple like this:

`You can also purchase CloudQuery directly through the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-lowyuyay5a37s#pdp-pricing).`

Reply:
Yeah that looks good!